instrumentation/opentelemetry-instrumentation-annotations/opentelemetry-instrumentation-annotations-incubator/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/instrumentationannotations/incubator/counted/CountedExample.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.instrumentationannotations.incubator.counted;
+
+import io.opentelemetry.instrumentation.annotations.incubator.Attribute;
+import io.opentelemetry.instrumentation.annotations.incubator.Counted;
+import io.opentelemetry.instrumentation.annotations.incubator.ReturnValueAttribute;
+import io.opentelemetry.instrumentation.annotations.incubator.StaticAttribute;
+import java.util.concurrent.CompletableFuture;
+
+public class CountedExample {
+
+  public static final String METRIC_NAME = "name.count";
+  public static final String METRIC_DESCRIPTION = "I am the description.";
+  public static final String METRIC_UNIT = "ms";
+  public static final String TO_STRING = "I am a to string object.";
+
+  @Counted(METRIC_NAME)
+  public void exampleWithName() {}
+
+  @Counted(value = "example.with.description.count", description = METRIC_DESCRIPTION)
```

Comment:
`example.with.description.count` is actually the name of the metric.  Will it be more appropriate to use `name="example.with.description.count"`?

Reply:
That's how @WithSpan works, I think we can keep it the same.

Reply:
@Duncan-tree-zhou just checking, are you going to make this change?

Reply:
Hey @trask , yes, I can make the change on Saturday

Reply:
The changes are applied, change annotation attribute `value` to `name` for @Count, @Time and @Attribute. cc @trask @fmhwong @brunobat.